main/guides/js-programming/notifiers.md

```diff
@@ -47,8 +47,7 @@ The key difference between the two is
 - `subscriptions` are lossless.
    - The consumer will see every value in the sequence.
 
-If your consumers only care about more recent states, use a `NotifierKit`.  For
-consumers that need to see all the values, use a `SubscriptionKit`. Subscriptions
+If your consumers only care about more recent states, use `makeNotifierKit()`. Notifiers
```

Comment:
These notifier / subscription changes look like more than typos / editorial. They look like (good) substantive fixes. A more clear commit message would be nice.